src/standalone/Soil/energy_hydrology.jl

```diff
@@ -696,3 +696,14 @@ function ClimaLSM.surface_height(model::EnergyHydrology{FT}, Y, p) where {FT}
     )
     return z_sfc
 end
+
+function ClimaLSM.get_drivers(model::EnergyHydrology)
+    bc = model.boundary_conditions.top
+    if typeof(bc) <: AtmosDrivenFluxBC{
```

Comment:
maybe this should have an else case too in case we want to run e.g. without a `PrescribedAtmos`

Reply:
good catch!!